### PR TITLE
Delete CassandraKeyValueServiceConfigManager

### PR DESCRIPTION
**Pull request by fsamuel-bs**
**Goals (and why)**: nobody uses this anymore — everybody initializes it with `.createSimpleManager`. Deleting it will make my next PR (live-reloadable CKVSConfig) easier as well.

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2886)

### CODE REVIEW

**Comment by fsamuel-bs** [comment]
This will break our internal code autobump. Fixing it is trivial though.

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java [hunk 1]
```diff
@@ -326,12 +323,12 @@ protected void initialize(boolean asyncInitialize) {
     }
 
     private void tryInitialize() {
-        boolean supportsCas = !configManager.getConfig().scyllaDb()
+        boolean supportsCas = !config.scyllaDb()
```
Comment (tboam) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java]: We support ScyllaDB?!?!

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlKeyValueService.java [hunk 2]
```diff
@@ -1060,9 +1051,13 @@ public void createTable(final TableReference tableRef, byte[] tableMetadata) {
         createTables(ImmutableMap.of(tableRef, tableMetadata));
     }
 
+    public static void main(String[] args) {
```
Comment (tboam) [inline atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlKeyValueService.java]: Don't think you meant to include this!

**Comment by fsamuel-bs** [comment]
@clockfort for SA.

File: atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java (hunk 1 above)
Comment (gsheasby) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java]: Only for @clockfort's nefarious side-projects...

Comment (clockfort) [reply atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java]: 😸 not officially, but once we move onto a CQL-only cassandra world it's probably a matter of time

**Pull request merged**